File: Python/UberVan.py
from car import car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterials = []

    def __init__(self, license, driver, typeCarAccepted, seatsMaterials):
        super(UberVan, self).__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterials = seatsMaterials

File: Java/UberVan.java
package Java;

class UberVan extends Car {
    String brand;
    String model;
    
    public UberVan(String license, Account driver, String brand, String model) {
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }

    @Override
    public void setPassenger(Integer passenger) {
        if (passenger == 6) {
        }else {
            System.out.println("Necesitas asignar 6 pasajeros");
        }
    }

    @Override
    void printDataCar() {
        super.printDataCar();
        System.out.println("Marca: " + brand + " Modelo: " + model);
    }
}
